downloads/swan-lake-release-notes/swan-lake-2201.0.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,381 @@
+     io:println(v1 is Employee); // Previously `true`, now `false`.
+   
+     Employee? v2 = empTable[0d]; // Lookup with decimal `0` as the key.
+     io:println(v2 is Employee); // Previously `true`, now `false`.
+   
+     Employee? v3 = empTable[0]; // Lookup with int `0` as the key.
+     io:println(v3 is Employee); // Only this will continue to be `true`.
+  }
+  ```
+
+To view bug fixes, see the [GitHub milestone for Swan Lake 2201.0.0](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+label%3ATeam%2FCompilerFE+milestone%3A%22Ballerina+2201.0.0%22).
+
+### Runtime Updates
+
+#### New Features
+
+#### Improvements
+
+#### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake 2201.0.0](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+2201.0.0%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+### Standard Library Updates
+
+#### New Features
+
+##### `graphql` Package
+- Added the CORS configuration support
+
+##### `http` Package
+
+- Implemented typed `headers` for the HTTP response
+- Added the `map<string>` data binding support for `application/www-x-form-urlencoded`
+- Added support to provide an inline request/response body with `x-form-urlencoded` content
+- Added compiler validation for the payload annotation usage
+
+#### Improvements
+
+##### `graphql` Package
+- Removed the deprecated `add` method in the `graphql:Context` object.
+
+##### `grpc` Package
+- Changed the `--proto_path` option of the gRPC CLI to `--proto-path`
+
+##### `websub` Package
+- Added the support for `readonly` parameters of remote methods
+
+#### `websubhub` Package
+- Added the support for `readonly` parameters of remote methods
+
+##### `kafka` Package
+- Made the `kafka:Caller` optional in the `onConsumerRecord` method of the `kafka:Service`
+- Allow the `readonly & kafka:ConsumerRecord[]` parameter type in the `onConsumerRecord` method
+
+#### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake 2201.0.0](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%22Swan+Lake+2201.0.0%22+label%3AType%2FBug).
+
+
+### Code to Cloud Updates
+
+#### New Features
+
+#### Improvements
+- The `awslambda` and `azure_functions` packages are no longer supported in single file projects
+
+#### Bug Fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake 2201.0.0 of the repositories below.
+
+- [C2C](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+2201.0.0%22)
+
+### Developer Tools Updates
+
+#### New Features
+
+##### Ballerina Shell
+
+- Added the module auto-import feature to the Ballerina Shell
+- Added the import statement for a module, which has a reference without an import statement based on the user’s input
```

Comment:
Aren't these the same? @DamithaSenevirathne 

Shall we say
```suggestion
- Added the module auto-import feature to the Ballerina Shell. If the user code contains a module prefix for a module that has not been imported, the Ballerina Shells provides the option to import the module.

```

Reply:
yes, +1 to update this

Reply:
+1. More like the Shell provides an interactive prompt for automatically adding the missing imports